share/spice/skyscanner_flight_search/footer.handlebars

```diff
@@ -0,0 +1,8 @@
+<div class="fix">
```

Comment:
Yes, I noticed that too. I did it so that if they didn't fit the price age would go below. I added some extra margin-top to the fix div to separate the price from the dates. 

Do you think they should be on the same line?